phys2bids/utils.py

```diff
@@ -23,15 +23,13 @@ def check_input_ext(filename, ext):
     """
     Checks that the given file has the given extension
     """
-    if '.gz' in ext:
-        if filename[-len(ext):] != ext:
-            filename = filename + ext
-        return filename
-    else:
-        if ext[0] != '.':
-            ext = '.' + ext
+    if '.gz' in filename:
+        filename = filename[:-3]
```

Comment:
What if we maybe do:

```python
if filename.endswith('.gz'):
    filename = filename[:-3]
```

This might be a bit safer in the (unlikely) event that someone has a file like `file.py.gz.txt` (or something else equally weird).